src/app/config/config.ts

```diff
@@ -40,6 +40,7 @@ const isDev =
   basicVars.core.nodeEnv === Environment.Dev ||
   basicVars.core.nodeEnv === Environment.Test
 const nodeEnv = isDev ? basicVars.core.nodeEnv : Environment.Prod
+const isLocalDev = basicVars.core.localDev
```

Comment:
Could we have leveraged `basicVars.core.nodeEnv === Environment.Dev` instead of introducing a new `isLocalDev` variable? 

Hmm, actually not sure 🤔 . You could have a dev environment testing real twilio crendentials too, so they're different concerns. But then again, the same applies to `isLocalDev`, so maybe we need something very specific like `useMockTwilio` 🤔